diff --git a/FABulous/fabric_generator/gen_fabric/gen_tile.py b/FABulous/fabric_generator/gen_fabric/gen_tile.py
index 14d5c1d..0bc41b2 100644
--- a/FABulous/fabric_generator/gen_fabric/gen_tile.py
+++ b/FABulous/fabric_generator/gen_fabric/gen_tile.py
@@ -673,7 +673,7 @@ def generateSuperTile(
                 )
                 writer.addConnectionScalar(f"Tile_X{x}Y{y}_UserCLKo", indentLevel=1)
             if (
-                0 <= x - 1 < len(superTile.tileMap[y])
+                0 <= x < len(superTile.tileMap[y]) - 1
                 and superTile.tileMap[y][x - 1] is not None
             ):
