diff --git a/fabulous/fabric_generator/gen_fabric/gen_fabric.py b/fabulous/fabric_generator/gen_fabric/gen_fabric.py
index 87858963..2c9397a3 100644
--- a/fabulous/fabric_generator/gen_fabric/gen_fabric.py
+++ b/fabulous/fabric_generator/gen_fabric/gen_fabric.py
@@ -343,7 +343,7 @@ def generateFabric(writer: CodeGenerator, fabric: Fabric) -> None:
 
             writer.addNewLine()
             writer.addComment(
-                "tile IO port will get directly connected to top-level tile module",
+                "tile IO port will get directly connected to top-level tile module\n (* keep *)",
                 onNewLine=True,
                 indentLevel=0,
             )
